modules/default/newsfeed/newsfeed.js

```diff
@@ -135,7 +135,7 @@ Module.register("newsfeed", {
 			sourceTitle: item.sourceTitle,
 			publishDate: moment(new Date(item.pubdate)).fromNow(),
 			title: item.title,
-			description: item.description
+			description: this.isShowingDescription ? item.description : null
```

Comment:
On second though: Shouldnt this be rather checked in the njk-template like the other options too?

Reply:
Agree with @rejas.

Reply:
Changed it, definitely better. Didn't notice the config being passed into the template as well...